### ISSUE TITLE
JAWS does not announce combo box State checked/not checked

### ISSUE THREAD

**Issue by RubyZ10** [FreedomScientific/VFO-standards-support#671]
### Summary

> Brief description of the issue. Please list any specific steps.
> JAWS does not announce fluent ui combo box checked or not checked state, it is impossible to differentiate if a option is checked or not.
> 
> 1. Go to https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox
> 2. In "Basic uncontrolled ComboBox" section, tab into "Basic multi-select ComboBox", tab through the options

### Expected result

> JAWS should announce **Option A 1 of 10..., Option B 2 of 10 **not checked**...**

### Actual result

> JAWS announced **Option A 1 of 10..., Option B 2 of 10...**
It is impossible to know Option A is selected, Option B is not
### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

**Comment by JAWS-test** [comment]
Tested with Chrome and JAWS 2022 and JAWS 2021. Can confirm the error. It does not occur with NVDA.

**Comment by JAWS-test** [comment]
The problem does not occur with Firefox + JAWS 2022. However, it does not seem to be a Chrome error, since Accessibility Insights shows that the status (IsSelected) is correctly transmitted.
